### ISSUE TITLE
build with wa-js: Cannot find module './eventEmitter'

### ISSUE THREAD

**Issue by antirek** [wppconnect-team/wppconnect#1217]
## Description

try build project with fresh versions

`````
> tsc

node_modules/@wppconnect/wa-js/dist/index.d.ts(29,21): error TS2307: Cannot find module './eventEmitter' or its corresponding type declarations.
node_modules/@wppconnect/wa-js/dist/index.d.ts(36,305): error TS2307: Cannot find module './eventEmitter' or its corresponding type declarations.

`````
I see that non exist dist/eventEmmiter/index.d.ts  but it exist in src 

## Environment

- **WPPConnect version(s):** 1.14.5
- **WA-JS version(s):** 2.8.1
- **Browser:** Chrome 102
- **OS:** Ubuntu 20.04
- **Node version:** Node 16

**Comment by antirek** [comment]
is it webpack trouble or my hands? )))

**Comment by antirek** [comment]
with wa.js 2.8.2 work on

**Comment by edgardmessias** [comment]
Thanks for reporting @antirek , after see your issue I released 2.8.2 version of wa-js
